### PR TITLE
WTL-852 Add Terraform configuration

### PR DESCRIPTION
**Pull request by apgrucza**
Moving Terraform configuration from the private repository to here

### CODE REVIEW

File: infra/terraform/examples/okta_native/main.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module "cloudfront_auth_okta_native" {
+  source = "github.com/iress/cloudfront-auth//infra/terraform/modules/okta_native"
```
Comment (patrickherrera) [inline infra/terraform/examples/okta_native/main.tf:2]: nit: double slash in path

**Review by patrickherrera — APPROVED** [review]
There is a fair bit in there, but it looks good

**Review by RodneyMarsh — APPROVED** [review]

File: infra/terraform/examples/okta_native/main.tf (hunk 1 above)
Comment (apgrucza) [reply infra/terraform/examples/okta_native/main.tf:2]: This is intentional. The double slash syntax is the way to specify a path inside the repo: https://www.terraform.io/docs/language/modules/sources.html#modules-in-package-sub-directories

Comment (patrickherrera) [reply infra/terraform/examples/okta_native/main.tf:2]: Cool, I did wonder if it was intentional somehow but couldn't for the life of me figure out what it meant

**Pull request merged**